build/alpine/Dockerfile.builder

```diff
@@ -14,6 +14,9 @@ WORKDIR /
 #
 # Finally, we build docs for haskell-src-exts without hyperlinking enabled
 # to avoid a Haddock segfault. See https://github.com/haskell/haddock/issues/928
+#
+# Note: git, ncurses, sed are added here for historical reasons; since
+# roughly 2019-03-28, they are included in prebuilder as well.
```

Comment:
If they are in the prebuilder, why not remove them from here?

Reply:
only so that builder will work with old prebuilder, until everything has been updated.  (the point of builder is that it doesn't have to rebuild prebuilder every time it's built, right?)